scripts/results/statistics/plot_sensor_cluster_stats_eeg_channel.py

```diff
@@ -72,7 +72,7 @@
 cluster_stats = permutation_cluster_1samp_test(
     data, threshold=threshold, n_jobs=N_JOBS, verbose=True, tail=tail,
     step_down_p=0.05, connectivity=connectivity,
-    n_permutations=n_permutations)
+    n_permutations=n_permutations, seed=random_state)
```

Comment:
if our results are actually sensititve to this seed then we need to do more permutations.

Reply:
I don't expect them to be very sensitive, but it's good to fix it to move toward 100% reproducibility

Reply:
could it explain the differnece of results between yours and mainak?

Reply:
I doubt it, I think I still get different results even with `seed` set (but need to double-check)

Reply:
not really because this wasn't the file which was showing the difference. In the other one, the `random_seed` was already set.